common/static/common/js/utils/edx.utils.validate.js

```diff
@@ -133,14 +179,18 @@
                                 context,
                                 content,
                                 customMsg,
-                                liveValidationMsg;
+                                liveValidationMsg,
+                                isEmailValue,
+                                isUsernameValue;
 
                             _.each(tests, function(value, key) {
                                 if (!value) {
                                     label = _fn.validate.getLabel($el.attr('id'));
                                     customMsg = $el.data('errormsg-' + key) || false;
                                     liveValidationMsg =
                                         $('#' + $el.attr('id') + '-validation-error-msg').text() || false;
+                                    isEmailValue =  _fn.validate.email.regex.test($el.val());
+                                    isUsernameValue = _fn.validate.username.ascii_regex.test($el.val());
```

Comment:
Looks like `isUsernameValue` is not used in `getMessage` function. Please remove it if it is unused.

Reply:
Removed. @GlugovGrGlib please review again